code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -107,12 +120,14 @@
 		if(insert(AM) == -1) // limit reached
 			break
 
-/obj/structure/closet/proc/open()
-	if(opened || !can_open())
+/obj/structure/closet/proc/open(mob/living/user)
+	if(opened || !can_open(user))
 		return
 	playsound(loc, open_sound, 15, 1, -3)
 	opened = 1
-	density = 0
+	if(!dense_when_open)
+		density = 0
+	climb_time *= 0.5 //it's faster to climb onto an open thing

```

Comment:
So am I.
Anyway I'm making the assumption that the crate isn't super fucking tall but is just, like tall enough that the lid being down makes climbing onto it hard. It doesn't necessarily make sense, but hey.